docker/layer/grant.sql.tpl

```diff
@@ -0,0 +1,2 @@
+grant all privileges on ${DB_USER}.* to '${DB_USER}'@'%';
```

Comment:
shouldn't this be the on ${DB_NAME}.* to '${DB_USER}  i know they are one in the same here i believe

Reply:
Good catch!  Thanks.  I'll change this up.